xlgui/cover.py

```diff
@@ -1046,19 +1046,8 @@ def __init__(self, parent, track, search=None):
 
         self.cover_image_box = self.builder.get_object('cover_image_box')
 
-        self.loading_indicator = Gtk.Alignment()
-        self.loading_indicator.props.xalign = 0.5
-        self.loading_indicator.props.yalign = 0.5
-        self.loading_indicator.set_size_request(350, 350)
-        self.cover_image_box.pack_start(self.loading_indicator, True, True, 0)
-
-        try:
-            spinner = Gtk.Spinner()
-            spinner.set_size_request(100, 100)
-            spinner.start()
-            self.loading_indicator.add(spinner)
-        except AttributeError: # Older than GTK 2.20 and PyGTK 2.22
-            self.loading_indicator.add(Gtk.Label(label=_('Loading...')))
+        self.__stack = self.builder.get_object('__stack')
+        self.__stack_ready = self.builder.get_object('__stack_ready')
```

Comment:
Oh, I've chosen `__stack` because I prefer to have fields private where possible. And I did the same in .ui files because they might be migrated to GtkTemplate eventually. What do you think? Still change it?